compiler/src/iree/compiler/Codegen/Common/test/materialize_tuning_specs.mlir

```diff
@@ -11,7 +11,6 @@
 // Check that the final tuning spec is as expected when the user tuning spec is provided.
 
 // CHECK-LABEL: module @iree_linked_tuning_spec
-// CHECK-SAME:    iree_codegen.tuning_spec_with_default_entrypoint
```

Comment:
I think this effectively break this test. This is fine, but we shoulda add a TODO to fix it once the merging logic can handle foreach_match ops.